worker/pom.xml

```diff
@@ -86,14 +86,16 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>85%</minimum>
+                                            <minimum>80%</minimum>
                                         </limit>
                                     </limits>
                                 </rule>
                             </rules>
                             <excludes>
+                                <exclude>**/gov/cms/ab2d/worker/SpringBootApp.java</exclude>
                                 <!-- exclude the bluebutton package from coverage as it only has a stub -->
-                                <exclude>**/gov/cms/ab2d/worker/adapter/bluebutton/*</exclude>
+                                <exclude>**/gov/cms/ab2d/worker/adapter/bluebutton/**</exclude>
+                                <exclude>**/gov/cms/ab2d/worker/service/FileService.java</exclude>
```

Comment:
We can test `FileService` without mocking.

Reply:
Removed FileService from the excludes list here.